model/integration.go

```diff
@@ -94,7 +113,8 @@ func (s Credentials) Validate() error {
 			validation.When(s.Type == CredentialTypeAWS, validation.Required,
 				validation.By(validateHostname))),
 		validation.Field(&s.DevicePolicyDocument,
-			validation.When(s.Type == CredentialTypeAWS, validation.Required)),
+			validation.When(s.Type == CredentialTypeAWS, validation.Required,
+				validation.By(validatePolicy))),
```

Comment:
I do not fully understand, `DevicePolicyDocument` is `*string`, so this: [return err](https://github.com/mendersoftware/iot-manager/pull/63/commits/3a46b8bba247e4b176dbd6c45a3af419c774374e#diff-8facb938c8871c2d5008b83d852f5dd59214a0032b6b76e56472ca77794af3d4R100) in `validatePolicy` is a dead code, right? is it because `validation` has to get a func that accepts `interface{}`? if there is no other way, ignore this comment.

Reply:
that's how the validation library works, I cannot do otherwise.